webpack.config.js

```diff
@@ -374,6 +422,8 @@ module.exports = {
       new SuppressChunksPlugin([
           { name: 'clarity-ui/clarity-ui', match: /\.js(\.map)?$/ },
           { name: 'clarity-ui/clarity-ui.min', match: /\.js(\.map)?$/ },
+          { name: 'clarity-ui/clarity-ui-dark', match: /\.js(\.map)?$/ },
```

Comment:
Is this going to add another file to the dev server that is 404?

Reply:
Yes.